packages/angular/cli/BUILD.bazel

```diff
@@ -61,7 +56,7 @@ ts_library(
         exclude = [
             # NB: we need to exclude the nested node_modules that is laid out by yarn workspaces
             "node_modules/**",
-            "cli/lib/config/workspace-schema.json",
```

Comment:
@alan-agius4 This path was wrong so `lib/config/workspace-schema.json` was _not_ actually being excluded. this dates back to your March commit https://github.com/angular/angular-cli/commit/4b0223b64ee091e4bd65b97e7867bb83afefa950. Is it correct to exclude it from this ts_library?

Reply:
Note that this file gets converted to `schema.json` which is included.

Reply:
Yeah it should be excluded.